test/livebook_web/live/hub/new_live_test.exs

```diff
@@ -112,14 +108,30 @@ defmodule LivebookWeb.Hub.NewLiveTest do
       # check if the page redirected to edit hub page
       # and check the flash message
       %{"success" => "Hub added successfully"} =
-        assert_redirect(view, path, @check_completion_data_interval)
+        assert_redirect(view, "/hub/team-#{name}?show-key=true", check_completion_data_interval())
+
+      # access the page and shows the teams key modal
+      {:ok, view, _html} = live(conn, "/hub/team-#{name}?show-key=true")
+      assert has_element?(view, "#show-key-modal")
+
+      # access the page when closes the modal
+      assert {:ok, view, _html} = live(conn, "/hub/team-#{name}")
+      refute has_element?(view, "#show-key-modal")
 
       # checks if the hub is in the sidebar
-      {:ok, view, _html} = live(conn, path)
-      hubs_html = view |> element("#hubs") |> render()
-      assert hubs_html =~ "🐈"
-      assert hubs_html =~ path
-      assert hubs_html =~ name
+      assert_hub(view, "/hub/team-#{name}", name)
     end
   end
+
+  defp check_completion_data_interval() do
```

Comment:
This time was not enough in my machine. But I don’t think we need to be conservative. This is the maximum time and not sleep time, so it can be a high value like 2 seconds.